deployment/entity-service/templates/worker-autoscale.yaml

```diff
@@ -2,13 +2,15 @@
 apiVersion: autoscaling/v1
 kind: HorizontalPodAutoscaler
 metadata:
+  labels:
+    {{- include "es.release_labels" . | indent 4 }}
   name: worker-auto-scale
 spec:
   scaleTargetRef:
     apiVersion: extensions/v1beta1
     kind: Deployment
     name: {{ .Release.Name }}-worker
   minReplicas: {{ required "workers.replicaCount must be provided."  .Values.workers.replicaCount }}
-  maxReplicas: 20
+  maxReplicas: {{ required "workers.autoscaler.maxReplicas must be provided." .Values.workers.autoscaler.maxReplicas }}
```

Comment:
**\<nitpick mode ON\>** 
the two parameters `minReplicas` and `maxReplicas` are clearly related. Why do we set them at different levels in the `Values` file? 
**\<upgrade to double nitpick mode NOW\>** 
Why do we set the `minReplicas` value as `replicaCount`? Wouldn't that just confuse some random DevOps guy when (s)he tries to configure the autoscaler?/
**\<all nitpick modes OFF\>**

Reply:
We set them at different levels because `replicaCount` is always required (whether the autoscaler is enabled or not), and I just decided that if the autoscaler is enabled we would use the initial number of replicas as the minimum. I'll add another configurable: `workers.autoscaler.minReplicas`.